coalib/settings/Setting.py

```diff
@@ -163,7 +163,11 @@ def __path__(self, origin=None, glob_escape_origin=False):
         :raises ValueError:        If no origin is specified in the setting
                                    nor the given origin parameter.
         """
-        strrep = str(self).strip()
+        if hasattr(self, 'value'):
+            strrep = self.value.strip()
```

Comment:
`str(self)` basically does [`unescape(self.value)`](https://gitlab.com/coala/coala-utils/blob/master/coala_utils/string_processing/StringConverter.py#L38). [`unescape`](https://gitlab.com/coala/coala-utils/blob/master/coala_utils/string_processing/Core.py#L409) basically strips away all backslashes. It looks like Lasse added the `str(self)` thing in the [first commit](https://github.com/coala/coala/commit/fbe61543788c20c0806755e27662ba48d4cf5d03#diff-2663e4c825081e483f6ed8d673b78545R26) that added `Settings.py`. I'm not entirely sure why we have that.

Anyway, I think the solution should be to override the `__str__` of `StringConvertor` here (by adding a `def __str__(string):`.